site/src/pages/UserSettingsPage/SecurityPage/SingleSignOnSection.tsx

```diff
@@ -30,7 +30,14 @@ export const redirectToOIDCAuth = (
   stateString: string,
   redirectTo: string,
 ) => {
-  window.location.href = `/api/v2/users/${toType}/callback?oidc_merge_state=${stateString}&redirect=${redirectTo}`
+  switch (toType) {
+    case "github":
+      return (window.location.href = `/api/v2/users/oauth2/github/callback?oidc_merge_state=${stateString}&redirect=${redirectTo}`)
```

Comment:
It's kinda weird to return an assignment statement, what about assigning and then returning on the next line?

Reply:
Will do that :+1:

Reply:
tbh I wasn't sure if I even needed a "return", but my IDE did that so the switch statements do not fallthrough

Reply:
I just added breaks to break the switch statement. No returns